packages/components/src/styles/__snapshots__/colors.spec.ts.snap

```diff
@@ -4,54 +4,80 @@ exports[`getThemedColors() should return correct colors for theme: dark 1`] = `
 {
   "activeColor": "rgba(126, 127, 130, 0.20)",
   "backgroundColor": "#0E0E12",
+  "backgroundColorDarken": "#000000",
+  "backgroundColorLighten": "#292934",
   "backgroundSurfaceColor": "#212225",
+  "backgroundSurfaceColorDarken": "#040405",
+  "backgroundSurfaceColorLighten": "#3e4045",
   "contrastHighColor": "#AFB0B3",
   "contrastHighColorDarken": "#909195",
+  "contrastHighColorLighten": "#cecfd1",
   "contrastLowColor": "#404044",
   "contrastMediumColor": "#7E7F82",
   "disabledColor": "#7E7F82",
   "errorColor": "#CB3333",
   "errorColorDarken": "#9a2727",
   "errorSoftColor": "#290A0A",
+  "errorSoftColorDarken": "#0d0808",
+  "errorSoftColorLighten": "#331f1f",
   "focusColor": "#0A0AFF",
   "hoverColor": "rgba(126, 127, 130, 0.20)",
   "hoverColorDarken": "#606163",
   "infoColor": "#027FFC",
   "infoSoftColor": "#001A33",
+  "infoSoftColorDarken": "#040a11",
+  "infoSoftColorLighten": "#0f2943",
   "primaryColor": "#FBFCFF",
   "primaryColorDarken": "#beceff",
   "successColor": "#00C77E",
   "successColorDarken": "#008a57",
   "successSoftColor": "#003320",
+  "successSoftColorDarken": "#04110c",
```

Comment:
Minor thing.. but why are new colors lower case when old colors are not?

Reply:
@denyo all the colors, which are generated with 'tinycolor2', have lower case.. I added `toUpperCase()` when generating and made them all to be uppercase